src/test/run-pass/command-exec.rs

```diff
@@ -84,4 +94,14 @@ fn main() {
     assert!(output.status.success());
     assert!(output.stderr.is_empty());
     assert_eq!(output.stdout, b"passed\n");
+
+    let output = Command::new(&me).arg("exec-test6").output().unwrap();
```

Comment:
Are exec-test6 and exec-test7 identical?

Reply:
I think morally yeah, they're just testing the two routes you can get at removing `PATH` from env (either by clearing or singularly removing it)

Reply:
Oh I missed the path manipulation up above.